src/library/scala/annotation/apiStatus.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala.annotation
+
+import scala.annotation.meta._
+
+/**
+ * An annotation to denote the API status.
+ *
+ * @param  message the advisory to print during compilation
+ * @param  category a string identifying the categorization of the restriction
+ * @param  since a string identifying the first version in which the status is applied
+ * @param  defaultAction the default severity of the restriction when the annotee is referenced
+ * @since  2.13.2
+ * @see    [[scala.annotation.apiStatus.Action]]
+ * @see    [[scala.annotation.apiStatus.Category]]
+ */
+@getter @setter @beanGetter @beanSetter @companionClass @companionMethod
+class apiStatus(
+  message: String,
+  category: String,
```

Comment:
maybe this should be just called "status", 1 to avoid introducing yet another term to the problem space here, and 2 to avoid overlap with `-Wconf`'s "category"

Reply:
Ah, you're actually integrating with the Wconf categories! Then that makes sense, I'll think about this more.